srcpkgs/dkms/template

```diff
@@ -21,14 +21,19 @@ if [ "$CROSS_BUILD" ]; then
 	depends+=" openssl-devel gmp-devel libada-devel libmpc-devel flex"
 fi
 
+do_build() {
+	make dkms dkms.8
+	vsed -i kernel_prerm.d_dkms \
+		-e 's-/usr/sbin-/usr/bin-g; s/^inst_kern=.*$/inst_kern="$2"/'
```

Comment:
Just keep the version we have in `$FILESDIR`. Our script is a bit more verbose (listing the kernel package name) and explicitly calling `/usr/bin/dkms` is better than calling `dkms` when the script verifies that the former exists but the latter is subject to `$PATH` shenanigans.

This will also make the script less likely to subtly break in a future update if there are unnoticed changes.

Reply:
Done. I've also made our script a bit more efficient, and also POSIX-compatible.